commune_app/tests/test_Expense_model.py

```diff
@@ -0,0 +1,33 @@
+from ..models import Expense, User
+import pytest
+
+
+class TestExpenseModel:
+    @pytest.mark.django_db
+    def test_new_expense(self):
+        expense = Expense(title="buying some food", budget="150", date="2011-09-01T13:20:30+03:00", assign=User(first_name="Faez", last_name="Zangariya", email="faezzangariyacs@gmail.com", phone_number="+972523324193"))
+        assert expense.title == "buying some food"
+        assert expense.budget == "150"
```

Comment:
I think it should be a number(int) not a string
```suggestion
        assert expense.budget == 150
```